components/banner/Banner.js

```diff
@@ -35,7 +26,7 @@ class Banner extends PureComponent {
               className={theme['close-button']}
               icon={<IconCloseMediumOutline />}
               color={this.getCloseButtonColor()}
-              onClick={this.handleClick}
+              onClick={onClose}
```

Comment:
shoudn't this one be in the destructuring?

Reply:
I think this is already the case?

const { children, className, icon, `onClose`, fullWidth, ...others } = this.props;

Reply:
@duivvv it is, you probably looked at the props of getCloseButtonColor? 😄

Reply:
Ah, the diff got me confused :)